### ISSUE TITLE
Can we have a get_handler(path)

### ISSUE THREAD

**Issue by halehawk** [constantinpape/z5#162]
I saw you modified your API and functions, thus we have to pass in a filehandler when we create a group or database. Then for my c wrapper, your filehandler which is a c++ object is hard to be used in users' c codes. Is it possible that you add a get_handler(c_str path) in your handler.cxx, thus I can use path info to get back handler and pass it to create group function call in my *.cc codes?

**Comment by constantinpape** [comment]
Yes, that should be possible. I could add a function 
```cpp
z5::filesystem::handle getFilesystemHandle(c_str path) {
   ...
}
```
to https://github.com/constantinpape/z5/blob/master/include/z5/filesystem/handle.hxx.
Would that help?

**Comment by halehawk** [comment]
It would be great if you can add it.

Thank

**Comment by constantinpape** [comment]
I have started to implement this in #165. @halehawk could you please check if this would work for you?

**Comment by halehawk** [comment]
Yeah, it works for me.

Thanks

**Comment by constantinpape** [comment]
Fixed in #165
